bayesian/utils.py

```diff
@@ -340,15 +340,21 @@ def build_nested_schema_dict(schema_dict):
     return result
 
 
-def get_next_component_from_graph(ecosystem, user_id):
+def get_next_component_from_graph(ecosystem, user_id, company):
```

Comment:
maybe `company=None`, because you might not get company decoded if its anoffline token or minimalist token